dist/docker-images/ziti-router/compose.yml

```diff
@@ -33,10 +32,8 @@ services:
       ZITI_BOOTSTRAP: true             # bootstrap the router if "true"
       ZITI_BOOTSTRAP_CONFIG: true      # make config file from env vars and defaults if "true," overwrite if "force"; requires ZITI_BOOTSTRAP=true
       ZITI_BOOTSTRAP_ENROLLMENT: true  # enroll with controller if "true," overwrite if "force"; requires ZITI_BOOTSTRAP=true
+      ZITI_AUTO_RENEW_CERTS: true      # renew certs every startup
       ZITI_ROUTER_TYPE: edge           # edge, fabric
-      ZITI_ROUTER_NAME: router         # identity files basename
```

Comment:
organize more vars into the image if they rarely need to be overridden to conserve noise in compose file